src/modules/predefinedDimensions.js

```diff
@@ -10,7 +10,7 @@ export const DIMENSION_ID_ORGUNIT = 'ou'
 export const DIMENSION_ID_ASSIGNED_CATEGORIES = 'co'
 export const DIMENSION_PROP_NO_ITEMS = 'noItems'
 
-const FIXED_DIMENSIONS = {
+const getFixedDimensionsInternal = () => ({
```

Comment:
Nice that you managed to solve the breaking change issue! 👍  I assume that just exporting this function directly (instead of routing through an "Internal" function didn't work? E.g. 
`export const getFixedDimensions = () ({
[DIMENSION_ID_DATA]: { ..... } 
})`

Reply:
all the "internal" functions are used more than once

Reply:
yes, but the "external" functions are just pointing to the "internal" directly, so couldn't the "internal" ones just be exported as well? (and omit the "internal" name)